logstash-core/src/main/java/org/logstash/instrument/witness/EventsWitness.java

```diff
@@ -124,6 +132,59 @@ public void queuePushDuration(long durationToAdd) {
         dirty = true;
     }
 
+    @Override
+    public String asJson() throws IOException {
+        return dirty ? SerializableWitness.super.asJson() : "";
+    }
+
+    @Override
+    public void genJson(final JsonGenerator gen, SerializerProvider provider) throws IOException {
+        new Serializer().innerSerialize(this, gen, provider);
```

Comment:
@jakelandis does `Serializer` have any state? It looks like we can just make it a constant right?